pkg/bastion/shell.go

```diff
@@ -828,12 +828,14 @@ GLOBAL OPTIONS:
 						}
 
 						var hosts []*dbmodels.Host
-						db = db.Preload("Groups")
 						if myself.HasRole("admin") {
-							db = db.Preload("SSHKey")
-						}
-						if err := dbmodels.HostsByIdentifiers(db, c.Args()).Find(&hosts).Error; err != nil {
-							return err
+							if err := dbmodels.HostsByIdentifiers(db.Preload("Groups").Preload("SSHKey"), c.Args()).Find(&hosts).Error; err != nil {
+								return err
```

Comment:
@jwessel We can wrap the error so we will have more context about where it happened and which func and etc.

Reply:
Your suggestion is good, however, @jwessel is only following the current project coding style, so I prefer to focus on what brings the PR and have a dedicated "good practice" issue/PR on the side